src/config.cc

```diff
@@ -71,6 +71,7 @@ void MainConfig::retrieve_config() {
   Singleton::Config::window()->default_size = {cfg.get<int>("default_window_size.width"), cfg.get<int>("default_window_size.height")};
   Singleton::Config::terminal()->make_command=cfg.get<std::string>("project.make_command");
   Singleton::Config::terminal()->cmake_command=cfg.get<std::string>("project.cmake_command");
+  Singleton::Config::terminal()->clang_format_command=cfg.get<std::string>("project.clang_format_command", "clang-format");
   Singleton::Config::terminal()->history_size=cfg.get<int>("terminal_history_size");
```

Comment:
Great! Also try to prioritize the remaining bugs, like the Windows dialogs if they are not yet working like the gtk-ones. I noticed for instance that one had to create a directory first instead of just writing the name of a new project when selecting New Project->C++ on Windows. Also, when running juci++ from msys-terminal, the home directory is not /home/[username], but /Users/... Maybe this is intended, not sure myself what is best here.